daemon-tests/tests/happy_path.rs

```diff
@@ -169,6 +169,9 @@ fn assert_eq_orders(mut published: CfdOrder, received: CfdOrder) {
     // align initial_funding_fee_per_lot so we can assert on the order
     published.initial_funding_fee_per_lot = received.initial_funding_fee_per_lot;
 
+    // align liquidation price so we can assert on the order
+    published.liquidation_price = received.liquidation_price;
```

Comment:
Note: I thought about if we can easily assert on the actual values, but it cannot easily be achieved and I don't think the actor tests are responsible for that so I did not invest time.